docs/using-the-jdbc-driver/using-plugins/UsingTheDeveloperPlugin.md

```diff
@@ -0,0 +1,43 @@
+## Developer Plugin
```

Comment:
Is this the name of the plugin? In the plugin list, its name is `dev` - is it worth renaming the doc or the plugin name so they match?

In case we expand the plugin functionality later, is it worth naming the plugin `devTools` or `developerTools` ?